bin/Powheg/run_pwg.py

```diff
@@ -20,17 +20,17 @@
 
 
 def runCommand(command, printIt = False, doIt = 1, TESTING = 0) :
-    if TESTING : 
```

Comment:
this file only has commit noise, I am afraid...
cfr https://stackoverflow.com/questions/21822650/disable-git-eol-conversions